x/auth/vesting/types/vesting_account_test.go

```diff
@@ -804,87 +803,21 @@ func TestPermanentLockedAccountMarshal(t *testing.T) {
 	require.NotNil(t, err)
 }
 
-func TestBaseVestingAccountMarshalJSONPB(t *testing.T) {
+func TestBaseVestingAccountMarshal(t *testing.T) {
```

Comment:
Remove this:
* https://github.com/line/lbm-sdk/pull/517/files#diff-79cdc833ea7de9c9dcb705ff51d3b0d2648ce639a7438c85e5a8cea2671577b1R807

Reply:
ditto